setup/freebsd/README.md

```diff
@@ -0,0 +1,28 @@
+# io.js Build FreeBSD Setup
+
+The current FreeBSD lives behind a VPN at [Voxer](http://voxer.com).
+If you want access, speak to [rvagg][rvagg] for further information.
```

Comment:
s/want/need, don't want to give Voxer the impression we're overly liberal about handing out access